commands/compile/compile.go

```diff
@@ -170,18 +170,14 @@ func Compile(ctx context.Context, req *rpc.CompileRequest, outStream, errStream
 		coreBuildCachePath = buildCachePath.Join("core")
 	}
 
-	requiredTools, err := pme.FindToolsRequiredForBuild(targetPlatform, buildPlatform)
-	if err != nil {
+	if _, err := pme.FindToolsRequiredForBuild(targetPlatform, buildPlatform); err != nil {
 		return nil, err
 	}
```

Comment:
Maybe this call can be completely removed

Reply:
I thought the same, but underneath if it doesn't find a tool release it fails. For this reason, I kept it